modules/apps/29-fee/types/msgs.go

```diff
@@ -41,3 +43,52 @@ func (msg MsgRegisterCounterpartyAddress) GetSigners() []sdk.AccAddress {
 	}
 	return []sdk.AccAddress{signer}
 }
+
+// NewMsgPayPacketFee creates a new instance of MsgPayPacketFee
+func NewMsgPayPacketFee(fee *Fee, sourcePortId, sourceChannelId, refundAccount string, relayers []string) *MsgPayPacketFee {
+	return &MsgPayPacketFee{
+		Fee:             fee,
+		SourcePortId:    sourcePortId,
+		SourceChannelId: sourceChannelId,
+		RefundAccount:   refundAccount,
+		Relayers:        relayers,
+	}
+}
+
+// ValidateBasic performs a basic check of the MsgPayPacketFee fields
+func (msg MsgPayPacketFee) ValidateBasic() error {
+	return nil
```

Comment:
Check that:

- Fee not empty (some fields may be empty but not all of them)
- PortID and channelID are valid. See 24-host
- Refund account not empty
- Enforce relayers list is nil for now. We can remove that check once we implement relayer-specific fees

Reply:
still need to check fee is not empty